app/customers_list.py

```diff
@@ -0,0 +1,18 @@
+from app.data.json_extraction import customers
+from app.objects.customer import Customer
+from app.objects.car import Car
+
+
+customers = [
+    Customer(
+        customer["name"],
+        customer["product_cart"],
+        customer["location"],
+        customer["money"],
+        Car(
+            customer["car"]["brand"],
+            customer["car"]["fuel_consumption"]
+        )
+    )
+    for customer in customers
+]
```

Comment:
pass object creation to main, we anyway have customer data, so we kinda dont need customers_list as a python module, when all responsibility is creating one list with customers.